### PR TITLE
:bug: Update migration wave form & modal styles

### PR DESCRIPTION
**Pull request by ibolton336**
Resolves https://issues.redhat.com/browse/MTA-742

### CODE REVIEW

File: client/src/app/pages/migration-waves/components/migration-wave-form.tsx [hunk 1]
```diff
@@ -289,71 +287,62 @@ export const WaveForm: React.FC<WaveFormProps> = ({
...
+                placeholder="MM/DD/YYYY"
+                value={value}
+                dateFormat={(val) => dayjs(val).format("MM/DD/YYYY")}
+                dateParse={(val) => dayjs(val).toDate()}
+                validators={[endDateRangeValidator]}
+                appendTo={() =>
+                  document.getElementById(
+                    "create-edit-migration-wave-modal"
+                  ) as HTMLElement
+                }
```
Comment (sjd78) [inline client/src/app/pages/migration-waves/components/migration-wave-form.tsx:340]:
```suggestion
                appendTo={() => document.body}
```

File: client/src/app/pages/migration-waves/components/migration-wave-form.tsx [hunk 2]
```diff
@@ -289,71 +287,62 @@ export const WaveForm: React.FC<WaveFormProps> = ({
...
+                placeholder="MM/DD/YYYY"
+                value={value}
+                dateFormat={(val) => dayjs(val).format("MM/DD/YYYY")}
+                dateParse={(val) => dayjs(val).toDate()}
+                validators={[startDateRangeValidator]}
+                appendTo={() =>
+                  document.getElementById(
+                    "create-edit-migration-wave-modal"
+                  ) as HTMLElement
+                }
```
Comment (sjd78) [inline client/src/app/pages/migration-waves/components/migration-wave-form.tsx:313]:
```suggestion
                appendTo={() => document.body}
```

... that should also help and just leave the modal height flickers to correspond to field error states taking up vertical space

**Review by sjd78 — CHANGES_REQUESTED** [review]
You lost the "to" between the two date inputs.  Not a deal breaker, but it is noticeable.

Changing the `appendTo` for the date input really does help.

**Comment by ibolton336** [comment]
This is a bug that @gitdallas would probably know how to best solve. Not sure the correct PF approach to make the modal /form field css be static when the dropdowns open. Seems like there is a bottom padding on the modal that changes when there is a new "last element" present within the modal scope in the dom.

**Comment by ibolton336** [comment]
> You lost the "to" between the two date inputs. Not a deal breaker, but it is noticeable.
> 
> Changing the `appendTo` for the date input really does help.

IMO the "to" seemed very out of place. But that is probably a @JustinXHale question.

File: client/src/app/pages/migration-waves/components/migration-wave-form.tsx (hunk 2 above)
Comment (ibolton336) [reply client/src/app/pages/migration-waves/components/migration-wave-form.tsx:313]: This works, thanks!

**Review by sjd78 — APPROVED** [review]
LGTM!

**Review by gitdallas — APPROVED** [review]

**Comment by JustinXHale** [comment]
> > You lost the "to" between the two date inputs. Not a deal breaker, but it is noticeable.
> > Changing the `appendTo` for the date input really does help.
> 
> IMO the "to" seemed very out of place. But that is probably a @JustinXHale question.

I agree it does seem out of place, but for PF we do use "to" centered for date and time range. [](https://www.patternfly.org/components/date-and-time/date-and-time-picker#date-and-time-range-picker )

**Pull request merged**